metron-interface/metron-alerts/protractor.conf.js

```diff
@@ -25,27 +25,28 @@ var SpecReporter = require('jasmine-spec-reporter').SpecReporter;
 exports.config = {
   allScriptsTimeout: 25000,
   specs: [
-    './e2e/login/login.e2e-spec.ts',
-    './e2e/alerts-list/alerts-list.e2e-spec.ts',
+    // './e2e/login/login.e2e-spec.ts',
```

Comment:
Similar comment to the above about removing vs commenting out code.

Reply:
Done